sass.py

```diff
@@ -448,6 +499,8 @@ def func_name(a, b):
             'not {1!r}'.format(SassFunction, custom_functions)
         )
 
+    importers = kwargs.pop('importers', None)
```

Comment:
let's do some validation of this parameter here.  We can also force them to be tuple of tuples and then the C code is safer (right now it'd crash if I pass in some other iterable type, let's test that too)